cabal-testsuite/PackageTests/Outdated/outdated_freeze.test.hs

```diff
@@ -1,21 +1,23 @@
 import Test.Cabal.Prelude
-main = cabalTest $ withRepo "repo"
-       $ forM_ ["--v2-freeze-file", "--freeze-file"] $ \arg -> do
+main = cabalTest $ withRepo "repo" $ do
+  forM_ ["--v2-freeze-file", "--freeze-file"] $ \arg -> do
+    cabal' "outdated" [arg] >>=
```

Comment:
Out of curiosity: why does the code use `>>=` instead `out <-  cabal' "outdated" [arg]`?

Reply:
Probably to avoid an additional `do` block, otherwise you would need to use name shadowing for the variable names.